core/drivers/pl022_spi.c

```diff
@@ -166,11 +166,9 @@ static enum spi_result pl022_txrx8(struct spi_chip *chip, uint8_t *wdat,
 	/* Capture remaining rdat not read above */
 	if (rdat) {
 		while ((j < num_pkts) &&
-			(io_read8(pd->base + SSPSR) & SSPSR_BSY))
-			if (io_read8(pd->base + SSPSR) & SSPSR_RNE) {
-				/* rx 1 packet */
-				rdat[j++] = io_read8(pd->base + SSPDR);
-			}
+			(io_read8(pd->base + SSPSR) & SSPSR_RNE))
```

Comment:
may I suggest to play with indentation and replace last tab above with 7 space chars and add `{}` since block is made of a comment and an instruction?



```
		while ((j < num_pkts) &&
		       (io_read8(pd->base + SSPSR) & SSPSR_RNE)) {
			/* rx 1 packet */
			rdat[j++] = io_read8(pd->base + SSPDR);
		}
```

Same comment for line 213 below.

Reply:
done